core/db/entities.py

```diff
@@ -1,19 +1,40 @@
 import uuid  # noqa
 
-from sqlalchemy import Column, String
+import sqlalchemy
```

Comment:
small thing but I find long module names distracting when using imports like this, what do you think about  `import sqlalchemy as sqla` or `sa`?

Reply:
I thought about the sqla option actually. Yeah can go for that